builtin/settingtypes.txt

```diff
@@ -142,7 +142,13 @@ screen_w (Screen width) int 1024 1 65535
 #    Height component of the initial window size. Ignored in fullscreen mode.
 screen_h (Screen height) int 600 1 65535
 
+#    Whether the window is maximized. Ignored in fullscreen mode.
+window_maximized (Window maximized) bool false
```

Comment:
This conflicts with `fullscreen`, hence I propose to combine them into an enum setting: "normal", "fullscreen", "maximized".

Reply:
Good point.
However the correct thing is not to combine the settings, but to not ignore it in fullscreen mode. If I resize/maximize/restore a window, then en- and disable fullscreen, the size and maximization is the same as it was before the fullscreening. Minetest should do the same (over multiple sessions).

Reply:
That being said, implementing this (especially also platform-independently) requires more work than I initially thought. So I won't do it now. (Maybe later, after some months.)
Logically, having separate settings still makes sense for the given reason.

I'll also remove the "Ignored in fullscreen mode." as that might be untrue in general. This does not imply that the settings are not ignored in fullscreen mode, it's just unspecified.